ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -813,6 +824,35 @@ export default {
         },
       },
     },
+    {
+      id: 'E12S Basic Relativity Yellow Hourglass',
+      // Orient where "Yellow" Anger Hourglass spawns
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '9824' }),
+      condition: (data, matches) => data.phase === 'basic',
+      preRun: (data, matches) => {
```

Comment:
Can you combine the preRun logic into infoText so that there doesn't need to be a `data.yellow` variable?